### ISSUE TITLE
Command setmoney

### ISSUE THREAD

**Issue by KingRyuShin** [ESX-Org/es_extended#206]
With the new update the admin command /setmoney ID TYPE AMOUNT doesn't work well..
it doesn't register the TYPE anymore
you can only do /setmoney ID AMOUNT
even tho the type is registered in the function

**Comment by ElPumpo** [comment]
Um it works just fine

**Comment by HumanTree92** [comment]
I'm having this problem as well. Not sure what is causing it.

**Comment by benzon** [comment]
This is an issue with current version :) it dos not register type it will just throw the invalid integer like the other 2 said....

**Comment by benzon** [comment]
But found the issue - essentialmode added it's own setmoney command facepalm

**Comment by BackrndSource** [comment]
> But found the issue - essentialmode added it's own setmoney command facepalm

es_admin2\server.lua

Line 371:
```
RegisterCommand('setmoney', function(source, args, raw)
```
Line 394:
```
RconPrint("Usage: setmoney [user-id] [money]\n")
```

@ElPumpo delete 'setmoney' command on es_extended and create 'setbankmoney' and 'setblackmoney' or rename 'setmoney' to something like 'setusermoney'?

**Comment by bsimser** [comment]
I think this issue needs a new PR. The PR that was included in #222 was closed and included the change to commands.lua to change line 94 from "setmoney" to "setusermoney" (in order to avoid the conflict with the same command in essentialmode). The current master branch doesn't have the change to commands.lua so it breaks.

**Comment by Nevos08** [comment]
Just comment out the command in es_admin2/server.lua and everything works fine :D
